### DISCUSSION TITLE
Please work on mali gpu

### DISCUSSION THREAD

**Discussion by Manishghorela** [skyline-emu/skyline#1766] [Game Requests]
Please work on mali gpu so that mali gpu can run pokemon let's go pikachu

**Comment by YapoOriginal** [comment]
I have the same problem where the emulator stays black when opening any game, I think it's because of the processor ;v, they couldn't solve it, it's a Matepad Pro 12.6 inch help me pls It's a Mali G78

**Reply by SansGamez** [reply]
Pls un Mali g52

**Comment by gamerm6777** [comment]
i also have the same problem my device is mali-g72

**Comment by Aungaung91** [comment]
Me too.😭😭😭Please make for Mali GPU Mali g77🥺🥺🥺

**Reply by Dekuzearou** [reply]
The devs are working hard to fix mali please just wait

**Comment by Crytonics** [comment]
Duplicate of #1789

**Comment by Marryjam** [comment]
Gup not supported in medeatek how can fix pls any one help me

**Reply by Crytonics** [reply]
Custom drivers are only available on Adreno gpu. You have Mali gpu, that's why it say that "Gpu is not supported" but that is only for custom drivers.

**Comment by parknich082** [comment]
stop crying about you alls cheap ass phones not working emulating a fucking nintendo switch, the devs will fix it when they fix it. this emulator is still new, theres more important stuff than making your alls shitty phone work. get over it

**Reply by Adg121991** [reply]
Please help mali user .. 🥺🥺🥺

**Comment by ZhakiThePro** [comment]
Ok me wait 😐

**Comment by Adg121991** [comment]
Thankyou
